### PR TITLE
fix: language-change-issue

### LINKED ISSUE #18156: [Media Library] Change the interface language of the Admin panel
### Required System information
- Node.js version: v16.14.0
- NPM version: v8.3.1
- Strapi version: v4.13.7
- Database: mysql
- Operating system: win10
- Is your project Javascript or Typescript: Javascript

### Describe the bug

After customizing the Admin panel to another language, such as Chinese, in the media library asset list, click Edit Asset, open the editing dialog box, press F5 to refresh the interface, and the Admin panel will become an English interface

### Steps to reproduce the behavior

1. Set the interface language to a language other than English on the profile page, and then save it
2. In the Media Library asset list, click Edit Asset, open the editing dialog box, press F5 to refresh the interface.The Admin panel has become an English interface

### PR DESCRIPTION
**Pull request by kunalvirk**
### What does it do?
Fix #18156

Describe the technical changes you did.
Removed the `usePersistentState` and made `lang` local to AssetPreview component.

### Why is it needed?

The media editor dialog was changing the `strapi-admin-language` to `en` which was overriding the locale chosen by the user.

### How to test it?

1. Change the language to anything other than english in `Profile` page and save
2. Then navigate to 'Media Library', select a preuploaded media and click `Edit asset`. Hit refresh, it should not change the language back to english

Thanks,

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by Boegie19 — APPROVED** [review]
@joshuaellis checked the PR fixes the issues only thing what I am not sure about is that is changes the srcLang of the track to be always English. so I am not sure if that could cause a problem.

**Comment by innerdvations** [comment]
Thanks for your PR! Could you please target the `develop` branch instead of `main`?

File: packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/AssetPreview.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const CardAsset = styled(Flex)`
 `;
 
 export const AssetPreview = forwardRef(({ mime, url, name, ...props }, ref) => {
-  const [lang] = usePersistentState('strapi-admin-language', 'en');
+  const lang = 'en';
```
Comment (joshuaellis) [inline packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/AssetPreview.js:18]: I actually think we can just get rid of this and the prop `srcLang` it's not really needed and iirc you need to know the language of the video, which might not be `en`.

**Review by joshuaellis — CHANGES_REQUESTED** [review]

**Comment by alexandrebodin** [comment]
> [!IMPORTANT] 
> With the release of Strapi 5, we have lifted the code freeze. To ensure we can effectively review and assist with contributions, we are now only accepting PRs targeting version 5 (the develop branch).<br/> 
> As your PR targets version 4, it has been automatically closed. However, we would greatly appreciate it if you could update your contribution to align with version 5 and resubmit!

**Pull request closed without merging**